packages/node/src/integrations/http.ts

```diff
@@ -96,22 +109,23 @@ function _createWrappedHandlerMaker(breadcrumbsEnabled: boolean, tracingEnabled:
         transaction = scope.getTransaction();
         if (transaction) {
           span = transaction.startChild({
-            description: `${typeof options === 'string' || !options.method ? 'GET' : options.method} ${requestUrl}`,
+            description: `${requestOptions.method || 'GET'} ${requestUrl}`,
             op: 'request',
           });
+          addSentryTraceHeader(span, requestOptions);
```

Comment:
I'd write it as:

```suggestion
          requestOptions = addSentryTraceHeader(requestOptions, span.toTraceparent());
```

Easier to test, no mutations, can pass any string value instead of being limited to just a `Span` instance.

Reply:
Or maybe event `requestOptions.headers = addSentryTraceHeader(requestOptions.headers, span.toTraceparent());` to be completely side-free (as what you do right now is accessing object as a reference, so it'll always modify the original value)

Reply:
At that point `addSentryTraceHeaer` just becomes the log statement, so I'll just move it all into the main function.

Reply:
Unresolving as no changes have been pushed to match the discussion above.

Reply:
Reresolving, as the change had been made - I just hadn't pushed it yet. But now I have.